sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/api/Azure.ResourceManager.RecoveryServicesBackup.netstandard2.0.cs

```diff
@@ -3309,6 +3014,57 @@ public partial class WorkloadItemResource : Azure.ResourceManager.Models.Tracked
+        public System.Collections.Generic.IDictionary<string, string> PropertyBag { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadJobTaskDetails> TasksList { get { throw null; } }
+    }
+    public partial class WorkloadJobTaskDetails
+    {
+        public WorkloadJobTaskDetails() { }
+        public string Status { get { throw null; } set { } }
+        public string TaskId { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct WorkloadOperationType : System.IEquatable<Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public WorkloadOperationType(string value) { throw null; }
+        public static Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType Invalid { get { throw null; } }
+        public static Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType Register { get { throw null; } }
+        public static Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType Reregister { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType left, Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType left, Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadOperationType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class WorkloadPointInTimeRecoveryPoint : Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadRecoveryPoint
+    {
+        public WorkloadPointInTimeRecoveryPoint() { }
+        public System.Collections.Generic.IList<Azure.ResourceManager.RecoveryServicesBackup.Models.PointInTimeRange> TimeRanges { get { throw null; } }
+    }
+    public partial class WorkloadPointInTimeRestoreRequest : Azure.ResourceManager.RecoveryServicesBackup.Models.WorkloadRestoreRequest
```

Comment:
avoid using `Request` as suffix, it seems that we have a lot of models that the name ends with `Request`.

Reply:
fixed